cpp-package/example/googlenet.cpp

```diff
@@ -22,9 +22,11 @@
 #include <string>
 #include <vector>
 #include <map>
+#include <fstream>
+#include "utils.h"
 #include "mxnet-cpp/MxNetCpp.h"
 
-
+using namespace std;
```

Comment:
explicitly use the std namespace with `std::` instead

Reply:
same with other occurrences

Reply:
@szha  addressed ur comments. if everything looks fine, please merge the PR